docs/competition/team-support/helpdesk.md

```diff
@@ -22,8 +22,8 @@ Helpdesk is the teams' primary source of support. Volunteers are expected to adv
  * Permanent marker
  * Large post-it notes
  * Radio
- * Emergency contact numbers for the Competition Team Support Coordinator (Rich Barlow) and the Competition Event Coordinator (Rob Spanton)
- * Spare robot badge for teams to borrow to try for size (must be clearly marked and returned to helpdesk ASAP)
+ * Emergency contact numbers
```

Comment:
For what purposes? For who?

Reply:
I guess the H&S Coordinator is the most important, everyone else can be radio-ed, Fixed in b7764ca